rcon/hooks.py

```diff
@@ -283,6 +276,47 @@ def update_player_steaminfo_on_connect(rcon, struct_log, steam_id_64):
         sess.commit()
 
 
+pendingTimers = {}
+
+
+@on_connected
+@inject_player_ids
+def notify_false_positives(rcon: RecordedRcon, _, name: str, steam_id_64: str):
+    c = get_config()["NOLEADER_AUTO_MOD"]
+    if not c["enabled"]:
+        logger.info("no leader auto mod is disabled")
+        return
+
+    if not name.endswith(" "):
+        return
+
+    logger.info("Detected player name with whitespace at the end: Warning him of false-positive events. Player name: "
+                + name)
+
+    def notify_player():
+        try:
+            rcon.do_message_player(steam_id_64=steam_id_64, message=c["whitespace_names_message"], by="CRcon", save_message=False)
+        except Exception as e:
+            logger.error("Could not message player " + name + "/" + steam_id_64, e)
+
+    # The player might not yet have finished connecting in order to send messages.
+    t = Timer(10, notify_player)
+    pendingTimers[steam_id_64] = t
+    t.start()
+
+
+@on_disconnected
+@inject_player_ids
+def cleanup_pending_timers(_, _1, _2, steam_id_64: str):
+    pt: Timer = pendingTimers.pop(steam_id_64, None)
```

Comment:
hmm, I would expect that this will just be a reference to an object after the timer run. Shouldn't be too much of an issue tbh, but maybe we should still have a regular timed job to cleanup all non-alive timers. wdym?